packages/patternfly-3/patternfly-react-extensions/less/classification-banner.less

```diff
@@ -1,10 +1,10 @@
 @classification-banner-height: 28px;
 @classification-banner-height-top: 3px;
-@classification-banner-padding-large: 42px;
-@classification-banner-padding-sml: 20px;
+@classification-banner-padding-lg: 42px;
+@classification-banner-padding-sm: 20px;
 
 .classification-banner-pf-banner{
-    background: @color-pf-black;
+    background: @color-pf-white;
```

Comment:
wrong change here, the comment was to replace line 9 `color: white` with `color: @pf-color-white`